01-spring-boot/README.md

```diff
@@ -96,4 +92,54 @@ Here are the operations that we will look to implement:
 This will allow you to test independently. 
 
 You can test your API from your IDE or install a free tool like [Restlet](https://chrome.google.com/webstore/detail/restlet-client-rest-api-t/aejoelaoggembcahagimdiliamlcdmfm?hl=en) 
-into your browser. 
\ No newline at end of file
+into your browser. 
+
+### Step 5 - Deploying with Docker
+
+In order to build a docker container we first need to test running the application as a jar locally. 
+There are two options for building locally:
+
+1. run `gradle build` from the command line (requires gradle to be installed)
+1. run `gradle build` from the IDE
+
+The following jar file should now be created: `build/libs/apiworkshop-0.0.1-SNAPSHOT.jar`
+You can run this jar using `java -jar build/libs/apiworkshop-0.0.1-SNAPSHOT.jar` the Spring gradle plugin will have packaged 
+everything for you to run your REST API. 
+
```

Comment:
Is it worth calling out that the app should be running at http://localhost:8080/hello to be tested with the REST client?